queue_services/entity-auth/pyproject.toml

```diff
@@ -28,8 +28,84 @@ Flask-Migrate = "^4.0.4"
 flask-babel = "^3.1.0"
 pytest = "^7.4.0"
 pytest-mock = "^3.11.1"
-black = "^23.7.0"
 freezegun = "^1.2.2"
+black = "^23.12.1"
+pylint = "^3.0.3"
+bandit = "^1.7.6"
+flake8-pyproject = "^1.2.3"
+isort = "^5.13.2"
+
+[tool.bandit]
+exclude_dirs = [".venv","tests"]
+skips = ["B104"]
+
+[tool.flake8]
+ignore = ["F401","E402", "Q000", "E203", "W503"]
```

Comment:
I have added W503 (line break before binary operator) to the ignore array as we make use of this convention in many places to break up long conditionals. Otherwise, we end up with lines of code that exceed the `max-line-length`, or we get warning W504 (line break after binary operator) if we split up the lines after the operator.